pom.xml

```diff
@@ -262,10 +262,10 @@
 							<version>3.4.0</version>
 						</driver>
 						<driver>
-							<name>edgedriver</name>
+							<name>internetexplorerdriver</name>
```

Comment:
Why do you change the order of the drivers? Please keep it as-is, so it is clear which versions are updated and which are added.